src/client/prodos/ethernet/drive/vedrive_high.asm

```diff
@@ -126,16 +126,18 @@ install_vedrive:
   ldx #>dev_not_found_msg
   jmp error_exit
 
+; calculate and save character to print for slot #
+: adc #'0'
+  pha
```

Comment:
At this point, we're taking the integer slot number, adding ASCII "`0`" to it in order to get a printable number, and then pushing that to the stack.  Since the shift lefts later send those high bits off to high bit heaven, the fact that we store to `io_base` what was a manipulated value in the accumulator doesn't change the actual value of `io_base`after this PR.